README.md

```diff
@@ -158,6 +158,38 @@ Example of using it in the controller:
     {
 ```
 
+# Route grouping
+You can add a `@LRDtags` parameter to the description of your controller to group the routes on the dashboard. Endpoint grouping is still done by routes and these tags will only be used as a friendly name for viewing the dashboard.
+@LRDTags is used to generate documentation in OpenAPI format
+Example of using it in the controller:
+
+```php
+    /**
+     * Class DemoController
+     * @package App\Http\Controllers
+     * @LRDtags Demonstration
```

Comment:
This `@LRDtags` if added to one controller, is picked up by all the other controllers too.
As example:

I only added this to the `api/v1/health` controller, while it goes on display to all the other controllers such as `api/v1/holi...`

![Screenshot 2024-06-03 at 10 51 28](https://github.com/rakutentech/laravel-request-docs/assets/629055/ed8b2368-2c3e-4c04-9612-8c36d4670e06)

```
/**
 * Class HealthController
 * Summary of method.
 * @LRDtags Demonstration
 * Demonstration management endpoint
 */
class HealthController extends ApiBaseController
{
    /**
     * Summary of method.
     * Description of method
     * @param HealthIndexRequest $request
     */
    public function index(HealthIndexRequest $request): Resource
```



Actually, I am not in favor of adding `@LRDtags`, the whole purpose of the lib is to not ask users to remember all the syntax sugar such as `@LRDtags` and it could generate the docs based on what a PHP user already knows. 

Summary and description work as intended and are fine!
Considering removal of `@LRDtags`, what do you think? @kitloong and @fernando-piovezan-instar